.spelling

```diff
@@ -125,3 +125,10 @@ Utility.app
 rc
 nginx.conf
 VirtualBox
+proxied
+Frontpage
```

Comment:
**Frontpage** is just the headline of `index.md`. This is never converted, since the frontpage is purely built by the template. But I had to add it prevent conflicts in this pull request though.

**proxied** is used in the `2013-03-16_how-to-fix-missing-x-forwarded-for-headers-for-qwebirc.md` post. It isn't recognized by `grunt-mdspell` even though it's spelled correct: https://en.wiktionary.org/wiki/proxied